onnx/test/test_backend_test.py

```diff
@@ -68,12 +66,10 @@ def prepare(
     def run_node(
         cls,
         node: NodeProto,
-        inputs: Any,
-        device: str = "CPU",
-        outputs_info: Sequence[tuple[numpy.dtype, tuple[int, ...]]] | None = None,
-        **kwargs: Any,  # noqa: ARG003
-    ) -> tuple[Any, ...] | None:
-        super().run_node(node, inputs, device=device, outputs_info=outputs_info)
+        *args: Any,  # noqa: ARG003
```

Comment:
I wonder why it was needed to change this piece of code.

Reply:
reverted